spec/relativetimeformat.html

```diff
@@ -79,16 +86,16 @@
             1. Let _result_ be Get(_patterns_, ToString(_v_)).
             1. Return _result_.
         1. If _v_ is less than 0, then
-          1. Let _tl_ be "past".
+          1. Let _tl_ be *"past"*.
         1. Else
-          1. Let _tl_ be "future".
+          1. Let _tl_ be *"future"*.
         1. Let _po_ be ? Get(_patterns_, _tl_).
-        1. Let _fv_ be _relativeTimeFormat_.[[NumberFormat]](_v_).
-        1. Let _pr_ be _relativeTimeFormat_.[[PluralRule]](_v_).
+        1. Let _fv_ be ? _relativeTimeFormat_.[[NumberFormat]].format(_v_).
```

Comment:
you have to get the method using `GetMethod`, and use `Call` to call it or, as I prefer, call the format method directly for numbers, the abstract operation: https://tc39.github.io/ecma402/#sec-formatnumber, e.g.:

```
1. Let _fv_ be ! FormatNumber(_relativeTimeFormat_.[[NumberFormat]], _v_).
```